### ISSUE TITLE
problems with pdb documentation

### ISSUE THREAD

**Issue by prjemian** [nexusformat/definitions#615]
Problems with reST syntax in PR #614 (and issue #553)

```
NeXus/definitions/build/manual/source/classes/base_classes/NXpdb.rst:50: SEVERE: Unexpected section title or transition.

=====================================================================================
NeXus/definitions/build/manual/source/classes/base_classes/NXpdb.rst:54: ERROR: Unexpected indentation.
NeXus/definitions/build/manual/source/classes/base_classes/NXpdb.rst:55: WARNING: Block quote ends without a blank line; unexpected unindent.
NeXus/definitions/build/manual/source/classes/base_classes/NXpdb.rst:68: SEVERE: Unexpected section title or transition.

=====================================================================================
NeXus/definitions/build/manual/source/classes/base_classes/NXpdb.rst:72: SEVERE: Unexpected section title or transition.

=====================================================================================
NeXus/definitions/build/manual/source/classes/base_classes/NXpdb.rst:86: SEVERE: Unexpected section title or transition.

=====================================================================================
NeXus/definitions/build/manual/source/classes/base_classes/NXpdb.rst:55: ERROR: Unknown target name: "loop".
```

**Comment by prjemian** [comment]
when showing computer code in reST, need to use "::" on preceding line of prose, then indent the computer code, such as this line of reST

```
the following code shows how to document computer code::

    print("Hello, world.")
```

which renders like this:

-----

the following code shows how to document computer code::

```
    print("Hello, world.")
```

**Comment by zjttoefs** [comment]
Since this will be merged when #614 is merged it can be closed.
